packages/react/src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, forwardRef, type Ref, Children, useId } from 'react';
+import { cx } from 'cva';
+import { ChevronDown } from '@obosbbl/grunnmuren-icons-react';
+import { Heading } from '../content';
+
+type AccordionProps = {
+  children: React.JSX.Element[] | React.JSX.Element;
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+};
+type AccordionItemProps = {
+  children: React.JSX.Element[];
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+};
+
+function Accordion(props: AccordionProps, ref: Ref<HTMLDivElement>) {
+  return (
+    <div
+      className={cx('flex flex-col rounded-lg bg-white p-2', props.className)}
```

Comment:
Her får vi hvit bakgrunn med avrundede hjørner når vi har en gruppe med accordions. Men hva om vi bare har en enkel? https://www.figma.com/file/9OvSg0ZXI5E1eQYi7AWiWn/Grunnmuren-2.0-%E2%94%82-Designsystem?node-id=2274%3A3326&mode=dev

Burde vi sjekke med UX om vi skal større single item accordions i det hele tatt? Ser at NAV for eksmpel har retningslinjer på at en Accordion aldri skal være alene https://aksel.nav.no/komponenter/core/accordion

Reply:
Osäker. Kan stötta att vi inte skall ha alene accordions, men vet feks att banken har det, så då må vi nog tänka på alternativa lösningar